common/lib/xmodule/xmodule/vertical_block.py

```diff
@@ -46,6 +50,15 @@ class VerticalBlock(SequenceFields, XModuleFields, StudioEditableBlock, XmlParse
 
     show_in_read_only_mode = True
 
+    discussion_enabled = Boolean(
+        display_name=_("Enable discussion for the Unit"),
+        help=_(
+            "Add discussion for the Unit."
+        ),
+        default=False,
+        scope=Scope.content,
+    )
```

Comment:
I think this should be moved to SequenceFields

Reply:
That makes sense this only had the logic and not the fields. I should have noted this earlier 😅

Reply:
@xitij2000 there is just one issue when I move this field to `SequenceFields` the MongoDB calls increase since now any Xblock that uses SequenceMixin will also have `dscussion_enabled` do you think creating another mixin to just add the discussion enabled field will be a better solution? Since then it will make way lesser call than now.

Reply:
I'm not sure, to be honest. Let's keep it here for now. I'd like to get feedback from someone who understands this better.